tox.ini

```
@@ -1,60 +1,35 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37,pypy,
-    py27-pyramid{12,13,14,15,16,17,18,19},
+    py34,py35,py36,py37,py38,pypy3,
```

Oops. I missed this one earlier.

```suggestion
    py34,py35,py36,py37,py38,py39,pypy3,
```

---

Yep, done. I was thinking the matrixing below would cover it, but I suppose this covers it Pyramid moves on. I've added this to the Travis file as well.